core/config/config.go

```diff
@@ -179,7 +215,10 @@ var (
 )
 
 var requiredFlags = []cli.Flag{
-	ConfigFileFlag,
+	EnvironmentFlag,
+	EthRpcUrlFlag,
+	EthWsUrlFlag,
+	AggregatorServerIpPortAddressFlag,
 	CredibleSquaringDeploymentFileFlag,
```

Comment:
remove this flag

Reply:
You mean move them from `requiredFlags` to `optionalFlags`